src/Catalyst.TestUtils/ContainerProvider.cs

```diff
@@ -116,10 +116,10 @@ public IConfigurationRoot ConfigurationRoot
             var keyRegistry = TestKeyRegistry.MockKeyRegistry();
             ContainerBuilder.RegisterInstance(keyRegistry).As<IKeyRegistry>();
 
-            ContainerBuilder.RegisterModule(new BulletProofsModule());
-            ContainerBuilder.RegisterModule(new KeystoreModule());
-            ContainerBuilder.RegisterModule(new KeySignerModule());
-            ContainerBuilder.RegisterModule(new HashingModule());
+            //ContainerBuilder.RegisterModule(new BulletProofsModule());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4345365)